<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>torrents</class>
 <widget class="QDialog" name="torrents">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KapTorrent</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Imagenes/drak.png</normaloff>:/Imagenes/drak.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 9pt &quot;Sans Serif&quot;;
font: bold;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="6">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Localizador de torrent</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Desde fichero</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Desde url</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>Desde magnet</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Â¿Donde guardar?</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_4">
        <property name="text">
         <string>Directorio</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>Insertar torrent a la cola</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/Imagenes/Correcto.png</normaloff>:/Imagenes/Correcto.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_7">
     <property name="text">
      <string>Eliminar torrent de la cola</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/Imagenes/bad.png</normaloff>:/Imagenes/bad.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="pushButton_6">
     <property name="text">
      <string>Procesar cola</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/Imagenes/good.png</normaloff>:/Imagenes/good.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>Torrent</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Guardar en...</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Datos de configuracion</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QPushButton" name="pushButton_4">
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/Imagenes/puerta.jpeg</normaloff>:/Imagenes/puerta.jpeg</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Opciones</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Sin limite de descarga</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QCheckBox" name="checkBox_4">
        <property name="text">
         <string>Sin limite de subida</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_5">
        <property name="text">
         <string>Verificar tras la descarga</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6" colspan="3">
       <widget class="QCheckBox" name="checkBox_6">
        <property name="text">
         <string>Activar adicional NAT/UPnP</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>Limitar descarga a...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Kb/s</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QCheckBox" name="checkBox_3">
        <property name="text">
         <string>Limitar subida a...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Kb/s</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="6">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Datos de la descarga</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Pares</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" colspan="5">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Descarga</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Kb/s</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Compartido</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Subida</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Kb/s</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="10">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Pasar a segundo plano</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
